MapboxNavigation/NavigationViewLayout.swift

```diff
@@ -25,9 +24,17 @@ extension NavigationView {
         wayNameView.bottomAnchor.constraint(equalTo: bottomBannerContainerView.topAnchor, constant: -10).isActive = true
         
         speedLimitView.topAnchor.constraint(equalTo: topBannerContainerView.bottomAnchor, constant: 10).isActive = true
-        speedLimitView.leadingAnchor.constraint(equalTo: safeLeadingAnchor, constant: 10).isActive = true
         speedLimitView.widthAnchor.constraint(equalToConstant: FloatingButton.buttonSize.width).isActive = true
         speedLimitView.heightAnchor.constraint(equalToConstant: FloatingButton.buttonSize.height).isActive = true
+        
+        switch floatingButtonsPosition {
+        case .topLeft:
+            floatingStackView.leadingAnchor.constraint(equalTo: safeLeadingAnchor, constant: 10).isActive = true
+            speedLimitView.trailingAnchor.constraint(equalTo: safeTrailingAnchor, constant: -10).isActive = true
+        default:
```

Comment:
Use `.topRight` here so that, if we ever add more cases to the enumeration, this switch statement will generate a warning about exhaustiveness so we know to update it.

Reply:
Thanks for the suggestion, I'll update the case name and remove the default